Assets/Scripts/SnapshotsController.cs

```diff
@@ -55,9 +55,9 @@ public void Init()
 
     public void SaveSnapshot(Pushpin pin, int pinIndex)
     {
-        PlayerPrefs.SetString(locationKey + pinIndex.ToString(), pin.LocationName);
-        PlayerPrefs.SetString(datetimeKey + pinIndex.ToString().ToString(), pin.SelectedDateTime.ToString());
-        PlayerPrefs.SetString(coordsKey + pinIndex.ToString().ToString(), pin.Location.ToString());
+        PlayerPrefs.SetString(locationKey + pinIndex, pin.LocationName);
```

Comment:
Probably a bad time for this comment but:

If students are sharing devices they will inherit each others pushpins.

Reply:
Ah yes. Well, I guess I'll add that to the "I expect comments about this" list :)